### PR TITLE
Implemented Census Registration Form Using HTML and CSS

### PR DESCRIPTION
**Pull request by Kenyai402**
This pull request includes the implementation of a simple registration form for data collection during a population census in Kenya, as per the assignment requirements. The following tasks were completed:

- Created a new branch (Kenyai402-web) as instructed.
- Developed a registration form using HTML and CSS to collect essential census data.
- Added hello.txt and updated README.md with details about the project and assignment workflow.
- Committed and pushed all changes to the Kenyai402-web branch.

Please review the form structure, design, and functionality to ensure it meets the assignment specifications.

### CODE REVIEW

File: index.html
Comment (WegenerSteven) [inline index.html:1]: This is good, implemented all the requirements and added more. Good work @Kenyai402. #1 #happycoding

File: style.css
Comment (WegenerSteven) [inline style.css:1]: The code has been written to it's standards and fully implemented the requirements. Good work @Kenyai402 .

**Review by WegenerSteven — APPROVED** [review]
Good work @Kenyai402

**Comment by Kenyai402** [comment]
Thaks for the feedback. I appreciate it.

**Pull request merged**